apis/cc-de_CH/atrium-api-cc-de_CH-common/src/main/kotlin/ch/tutteli/atrium/api/cc/de_CH/iterableAssertions.kt

```diff
@@ -150,9 +150,25 @@ fun <E: Any, T: Iterable<E?>> Assert<T>.enthaeltNullableEintraege(assertionCreat
  * @return This plant to support a fluent API.
  * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
  */
-fun <E : Any, T : Iterable<E>> Assert<T>.enthaeltStrikt(expected: E, vararg otherExpected: E): AssertionPlant<T>
+fun <E : Any, T : Iterable<E>> Assert<T>.enthaeltExakt(expected: E, vararg otherExpected: E): AssertionPlant<T>
     = enthaelt.inGegebenerReihenfolge.nur.werte(expected, *otherExpected)
 
+/**
+ * Makes the assertion that [AssertionPlant.subject] contains only [expected] and the [otherExpected] (if given) in
+ * the defined order.
+ *
+ * It is a shortcut for `enthaelt.inGegebenerReihenfolge.nur.werte(expected, *otherExpected)`
+ *
+ * @return This plant to support a fluent API.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ */
+@Deprecated(
+    message = "Replaced with enthaeltExakt for clearer naming",
+    replaceWith = ReplaceWith("enthaeltExakt(expected,otherExpected)", "ch.tutteli.atrium.api.cc.de_CH")
```

Comment:
- You forgot `*` in front of otherExpected. 
- good idea to add an import :+1: yet, either it requires `.*` or the function name as suffix (I think, cannot check it). Both are fine with me.

Reply:
Fixed! I used the function name in the import, as I generally prefer adding only the necessary stuff in the code :)